### DISCUSSION TITLE
Is there something like Webstom/Intellij postfix templates?

### DISCUSSION THREAD

**Discussion by pogopaule** [hrsh7th/nvim-cmp#855] [Q&A] [answered]
Jetbrain IDEs have this one feature I am missing in neovim: [Postfix Templates](https://www.jetbrains.com/help/webstorm/using-postfix-templates.html)

Is this already possible somehow? If not, could somebody point me into the direction where and how to implement it in nvim-cmp? I'd give it a try and might open a PR.

CoC already got it https://github.com/fannheyward/coc-postfix

**Comment by Samasaur1** [comment, marked as answer]
If you haven't figured it out yet, you can do this with LuaSnip:

https://github.com/L3MON4D3/LuaSnip/issues/529
